domains/authorization/Authorization.API/API/Models/ApplicationDbContext.cs

```diff
@@ -50,6 +52,14 @@ private static void ConfigureOrganizationTable(ModelBuilder modelBuilder)
         modelBuilder.Entity<Organization>().Property(o => o.TermsVersion);
 
         modelBuilder.Entity<Organization>().Property(o => o.TermsAcceptanceDate);
+
+        modelBuilder.Entity<Organization>().Property(o => o.ServiceProviderTermsAccepted)
+            .IsRequired()
+            .HasDefaultValue(false);
+
+        modelBuilder.Entity<Organization>().Property(o => o.ServiceProviderTermsVersion);
```

Comment:
This line does nothing. You don't need to specify each property on an object. Only the ones you want special treatment on :)

Reply:
They are all special to me

Reply:
I see what you mean. But for the sake of simplicity, i decided to mirror how it is done in Terms. I think it's fine this time.